src/models/db.js

```diff
@@ -77,6 +78,13 @@ export default function() {
       deferreds.pop().resolve(db);
     }
     state = READY;
+
+    // Load default permissions.
+    const permissions = config.get('permissions').map(permission => {
```

Comment:
That's what we are currently doing :) https://github.com/mozilla-sensorweb/sensorweb-server/pull/71/files/d791ffafbd6b3cbc898c242f35b9a2063c513768#diff-9d9a6cd82f41984872a66a3ab0d440c4R100